Pilko yöllinen raporttiajo kolmeen osaan

jarnovayrynen opened the pull request.

jarnovayrynen committed:
Suojele create index uudelleenajolta tuotannossa

jarnovayrynen committed:
Pilko päivitä-raportti-cachet kolmeen osaan

Ajetaan erillisinä eräajoina raportit:
-toteutuneet materiaalit
-pohjavesialueiden suolatoteumat
-toteumamäärät

Tavoite selvittää jatkuuko max_wal_size varoitukset myös muutosten jälkeen ja jos, mistä tarkalleen ne aiheutuvat.

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-5100

jarnovayrynen committed:
Päivitä testien tarvitsemat raporttipäivitykset

K1ll3rF0x commented on src/clj/harja/palvelin/raportointi.clj:
```diff
@@ -147,18 +147,40 @@
+                                           ;; otetaan 3h lukko, jotta varmasti voimassa ajon jälkeen
+                                           (* 60 180)))))
+
+(defn paivita_raportti_toteuma_maarat [db]
```
```suggestion
(defn paivita_raportti_toteuma_maarat! [db]
```

K1ll3rF0x commented on src/clj/harja/palvelin/raportointi.clj:
```diff
@@ -189,10 +211,17 @@
-    (assoc this :raportti-cache-ajastus (paivita-raportti-cache-oisin! db)))
+    (assoc this :toteutuneet-materiaalit-ajastus (paivita_raportti_toteutuneet_materiaalit! db)
+                :pohjavesialueiden-suolatoteumat-ajastus (paivita_raportti_pohjavesialueiden_suolatoteumat! db)
+                :toteuma-maarat-ajastus (paivita_raportti_toteuma_maarat db)))
```
```suggestion
                :toteuma-maarat-ajastus (paivita_raportti_toteuma_maarat! db)))
```

K1ll3rF0x reviewed (approved):
Yhden pienen huomatuksen laitoin funktion nimeen liittyen.

K1ll3rF0x commented:
Tämä backend-testi epäonnistuu:

lein test harja.kyselyt.suolasakko-test
{:result false, :seed 1629794370109, :failing-size 1, :num-tests 2, :fail [-6.3 -28.7 7 2390 7456], :shrunk {:total-nodes-visited 30, :depth 29, :result false, :smallest [0.0 0.0 1 1 1]}, :test-var "muuta-sakon-maaraa"}
lein test :only harja.kyselyt.suolasakko-test/muuta-sakon-maaraa
FAIL in (muuta-sakon-maaraa) (clojure_test.cljc:21)
expected: result
  actual: false

jarnovayrynen committed:
Oikeat tiedot suolasakkotestiin

jarnovayrynen committed:
Korjaa huutomerkki funktion nimeen

Laita ajastus minuutin yli puolen yön, siltä varalta ettei pvm-raja sotke asioita
